controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorGroupImpl.java

```diff
@@ -255,16 +257,16 @@ public void notifyProcessFailure(String process) throws CheckpointStoreException
 
             for (Map.Entry<String, Position> entry : map.entrySet()) {
                 // 1. Notify reader group about failed readers
-                log.info("{} Notifying readerOffline reader={}, position={}", this.objectId, entry.getKey(), entry.getValue());
+                log.debug("{} Notifying readerOffline reader={}, position={}", this.objectId, entry.getKey(), entry.getValue());
```

Comment:
let us keep this at info. this is important and rare. we need to know when and at what positions readers were deemed offline.

Reply:
Is this something a sysadmin would need to look at or understand? If not, it should be `debug`

Reply:
sure will  keep this at info.